docker/downloader.sh

```diff
@@ -0,0 +1,648 @@
+    echo "  scp tls.cert $REMOTE_USER@$EXTERNAL_IP:/home/$REMOTE_USER/.lnd/"
+    echo
+    read -p "Press enter after you have transferred the files " -r -n 1 </dev/tty
+    echo
+
+    while true; do
+      echo "Checking files..."
+
+      # check if the two files are present in the .lnd directory
+      if [ -f "$HOME/.lnd/admin.macaroon" ] && [ -f "$HOME/.lnd/tls.cert" ]; then
+        echo "All files look good..."
+        break
+      else
+        echo "Some files do not look correct. Make sure you put them both in your .lnd directory."
+        read -p "Press enter after you have transferred the files " -r -n 1 </dev/tty
+        continue
+      fi
+    done
+  fi
+  if [ "$USE_THUNDERHUB" = false ]; then
+    # Remove thunderhub from the docker-compose file
+    sed -i '/### START_OF_THUNDERHUB ###/,/### END_OF_THUNDERHUB ###/d' ./docker-compose.yaml
+
+    # Remove thunderhub data directory 'thunderhub_datadir:' from the docker-compose file
+    sed -i '/thunderhub_datadir:/d' ./docker-compose.yaml
+  fi
+else # Is Guardian
+  if [ "$USE_ESPLORA" = true ]; then
+    case $NETWORK_TYPE in
+    mainnet)
+      DEFAULT_ESPLORA_URL="https://blockstream.info/api"
+      ;;
+    testnet)
+      DEFAULT_ESPLORA_URL="https://blockstream.info/testnet/api"
+      ;;
+    signet)
+      DEFAULT_ESPLORA_URL="https://blockstream.info/signet/api"
+      ;;
+    mutinynet)
+      DEFAULT_ESPLORA_URL="https://mutinynet.com/api"
+      ;;
+    *)
+      echo "Error: Unknown network type for Esplora URL."
+      exit 1
+      ;;
+    esac
+
+    read -p "Enter the Esplora URL you'd like to use to get $NETWORK_TYPE blockdata from, or click enter to use the network default: [$DEFAULT_ESPLORA_URL]: " ESPLORA_URL </dev/tty
+    ESPLORA_URL=${ESPLORA_URL:-$DEFAULT_ESPLORA_URL}
+    echo "Using Esplora URL: $ESPLORA_URL"
+
+    # Set the esplora URL in the docker-compose file
+    sed -i "s|FM_BITCOIN_RPC_URL=https://blockstream.info/api/|FM_BITCOIN_RPC_URL=$ESPLORA_URL|" ./docker-compose.yaml
+
+    # Set the bitcoin network in the docker-compose file if not mutinynet default
+    if [ "$NETWORK_TYPE" != "mutinynet" ]; then
+      sed -i "s|FM_BITCOIN_NETWORK=signet|FM_BITCOIN_NETWORK=$NETWORK_TYPE|" ./docker-compose.yaml
+    fi
+
+  else # Using Bitcoin Node for Guardian blockchain data
+    echo "You will need to provide the Bitcoin Node RPC URL to get blockchain data."
+    read -p "Enter the RPC URL for your Bitcoin node (ex. https://mynode.m.voltageapp.io:8332): " -a bitcoin_rpc_url </dev/tty
+    if [[ -z ${bitcoin_rpc_url[*]} ]]; then
+      echo 'Error: You must set a Bitcoin RPC URL if you configure the guardian with an existing Bitcoin node' >&2
+      exit 1
+    fi
+    echo "Using Bitcoin RPC URL: $bitcoin_rpc_url"
+
+    # Set the bitcoin RPC URL in the docker-compose file
+    sed -i "s|FM_BITCOIN_RPC_URL=https://blockstream.info/api/|FM_BITCOIN_RPC_URL=$bitcoin_rpc_url|" ./docker-compose.yaml
+  fi
+fi
```

Comment:
Going through the esplora flow still launches the bitcoind docker container and starts syncing the chain. I think you should probably strip out or insert the bitcoind part of the docker-compose.yaml file based on the results here.